CRM/Core/BAO/UFGroup.php

```diff
@@ -2339,7 +2339,10 @@ public static function setProfileDefaults(
             $defaults[$name . '_custom'] = $details[$name . '_custom'];
           }
           elseif ($name == 'preferred_communication_method') {
-            $v = explode(CRM_Core_DAO::VALUE_SEPARATOR, $details[$name]);
+            $v = $details[$name];
+            if (!is_array($details[$name])) {
```

Comment:
I think it would be slightly cleaner to use $v in the IF - but not a blocker in any way